app/controllers/concerns/phone_confirmation_flow.rb

```diff
@@ -76,7 +76,7 @@ def send_confirmation_code
     job.perform_later(
       code: confirmation_code,
       phone: unconfirmed_phone,
-      otp_created_at: current_user.direct_otp_sent_at.to_s
+      otp_created_at: (current_user.direct_otp_sent_at || Time.zone.now).to_s
```

Comment:
thoughts on making this a default arg for the job itself rather than when we call the job?